roles/openshift_metrics/templates/route.j2

```diff
@@ -16,6 +16,10 @@ spec:
 {% if tls is defined %}
   tls:
     termination: {{ tls.termination }}
+{% if tls.ca_certificate is defined and tls.ca_certificate | length > 0%}
+    CACertificate: |
+{{ tls.ca_certificate|indent(6, true) }}
+{% endif %}
```

Comment:
@mwringe: do you see a problem in having separate files for the key and
cert/chain, instead of a single pem as we have in the deployer?